cts/cli/regression.tools.exp

```diff
@@ -5065,8 +5071,14 @@ Remaining: httpd-bundle-0 capacity:
 Remaining: httpd-bundle-1 capacity:
 Remaining: httpd-bundle-2 capacity:
 
-Start      httpd-bundle-2     (      cluster01 )  due to unrunnable httpd-bundle-docker-2 start (blocked)
-Start      httpd:2            ( httpd-bundle-2 )  due to unrunnable httpd-bundle-docker-2 start (blocked)
+Restart    httpd-bundle-docker-0     (      cluster01 )  due to resource definition change
+Restart    httpd-bundle-0            (      cluster01 )  due to required httpd-bundle-docker-0 start
+Restart    httpd:0                   ( httpd-bundle-0 )  due to required httpd-bundle-docker-0 start
+Restart    httpd-bundle-docker-1     (      cluster02 )  due to resource definition change
+Restart    httpd-bundle-1            (      cluster02 )  due to required httpd-bundle-docker-1 start
+Restart    httpd:1                   ( httpd-bundle-1 )  due to required httpd-bundle-docker-1 start
+Start      httpd-bundle-2            (      cluster01 )  due to unrunnable httpd-bundle-docker-2 start (blocked)
+Start      httpd:2                   ( httpd-bundle-2 )  due to unrunnable httpd-bundle-docker-2 start (blocked)
```

Comment:
These differences are not related to your changes. I'm not sure why you're seeing them. CI is failing though and I'm guessing it's not seeing them.

Reply:
Hm, weird.  Let's drop it and see what happens.